core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -95,9 +106,21 @@ private[data] sealed abstract class WriterTInstances extends WriterTInstances0 {
       implicit val F0: Monad[F] = F
       implicit val L0: Monoid[L] = L
     }
+
+  implicit def catsDataFoldableForWriterT[F[_], L, V](implicit F: Foldable[F]): Foldable[WriterT[F, L, ?]] =
```

Comment:
The `Traverse` instance is more specific and thus should have a higher priority than the `Foldable` instance. 
In case you are curious, here is the guideline in this regard. 
https://typelevel.org/cats/guidelines.html#a-idimplicit-priority-hrefimplicit-prioritya-implicit-instance-priority
I guess you can leave the instance of `Traverse` where you have now and move the `Foldable` instance down to `WriterTInstance1`

Reply:
thanks, makes perfectly sense. I wasn't sure about the precedence there, I should have read the guidelines myself. I'm changing it.